spec/audited/auditor_spec.rb

```diff
@@ -414,6 +414,45 @@ def non_column_attr=(val)
     end
   end
 
+  describe "on touch" do
+    before do
+      @user = create_user(name: "Brandon", status: :active, audit_comment: "Touch")
+    end
+
+    around do |example|
+      freeze_time do
```

Comment:
Looks like freezing won't work on old Rails, and different DBs are storing it differently. I'd suggest checking that it is [within](https://relishapp.com/rspec/rspec-expectations/v/3-12/docs/built-in-matchers/be-within-matcher) a second of Time.current instead.

Reply:
Good suggestion, updated!